### PR TITLE
Remove Snapp Binaries And Tests

### PR DESCRIPTION
**Pull request by nlordell**
This is the first of many PRs to remove Snapp from `dex-services`. Note that some of the old types in dfusion_core that are only used for Snapp are still there, I figure we can remove them a little at a time.

The last stable version of Snapp can be found here: https://github.com/gnosis/dex-services/tree/stable-snapp

### Test Plan

Ensure stableX CI still passes!

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -28,43 +28,6 @@ services:
...
-    depends_on:
-      - ganache-cli
-    volumes:
-      # Don't sync target, otherwise we rebuild on every deploy
-      - ./dex-contracts:/app/dex-contracts
-      - ./dfusion_rust_core/src:/app/dfusion_rust_core/src
-      - ./driver/src:/app/driver/src
-
-  graph-listener:
-    << : *rust-service
```
Comment (fleupold) [inline docker-compose.yml]: We might not need the rust-service thingy anymore either.

Comment (nlordell) [reply docker-compose.yml]: It's still used by the `stablex` docker compose service. I wanted to keep this "remove only" as much as possible. I imagine once we are done removing we can refactor the CI to not use docker compose anymore at all.

**Comment by nlordell** [comment]
Just waiting on #485 before merging

**Comment by bh2smith** [comment]
> Just waiting on #485 before merging

Wait no more! It be merged.

File: .travis.yml [hunk 2]
```diff
@@ -32,31 +32,9 @@ matrix:
...
-      install:
-        - sudo apt-get update && sudo apt-get install -y python-pip && sudo pip install awscli
-        - $(aws ecr get-login --no-include-email --region $AWS_REGION)
-        - docker pull $REGISTRY_URI:$TRAVIS_COMMIT
-        - docker tag $REGISTRY_URI:$TRAVIS_COMMIT stablex-binary
-        - (cd dex-contracts && yarn --frozen-lockfile)
-        - ./scripts/setup_contracts.sh
-      script:
+        # Build and publish compact image with compiled binary
+        - docker build --tag stablex-binary --build-arg SOLVER_BASE=163030813197.dkr.ecr.us-east-1.amazonaws.com/dex-solver:v0.5.8 --build-arg RUST_BASE=rust-binary -f docker/rust/Dockerfile .
```
Comment (bh2smith) [inline .travis.yml]: Are we still doing this?

**Pull request merged**